language/README.md

```diff
@@ -0,0 +1,11 @@
+# Cloud Natural Language API Go samples
+
+This directory contains [Cloud Natural Language API](https://cloud.google.com/language/) Go samples.
```

Comment:
/natural-language/
I think it's either a branding thing or an seo thing. I think the idea is, for urls and such, use the full name, but for code (eg package names) it's okay to just use 'language'